include/deal.II/physics/transformations.h

```diff
@@ -789,6 +857,55 @@ namespace internal
 
 
 
+template<typename Number>
+Tensor<2,2,Number>
+Physics::Transformations::Angular::rotation_matrix_2d (const Number &angle)
+{
+  return Tensor<2,2> (
+  {
+    {
```

Comment:
This requires C++11. But you can make it compatible with C++98 if you just declare a 2x2 local array and pass that as explicit argument to the constructor.